packages/dds/tree/src/schema-stored/Stored and View Schema.md

```diff
@@ -35,104 +34,116 @@ Places we can store stored schema information:
 
 -   Inline in whatever place is referring to it (see list below)
 -   As stored data
--   Hard coded into shared tree (constraints like the tree being a tree not a DAG fall into this)
+-   Hard coded into shared-tree TODO: SharedTree? (constraints like the tree being a tree not a DAG fall into this)
```

Comment:
```suggestion
-   Hard coded into SharedTree (constraints like the tree being a tree not a DAG fall into this)
```
Is this correct? I'm not sure if we're trying to refer to the actual DDS or the package.

Reply:
this is really trying to cover schema as source code for all the cases not covered in the below bullet point.
Since this is about theoretical places and architectures and not our concreate implementation, I'm temptyed to avoid real class names (ex: SharedTreeCore or SharedTree) here.

Maybe this would be better:
```suggestion
-   Hard coded into the shared-tree DDS (constraints like the tree being a tree instead of a DAG fall into this)
```